app/build.gradle

```diff
@@ -30,4 +30,5 @@ dependencies {
     compile 'com.android.support:cardview-v7:23.1.1'
     compile 'com.android.support:recyclerview-v7:23.1.1'
     compile 'com.jakewharton:butterknife:7.0.1'
+    compile 'com.android.support:support-v4:23.1.1'
```

Comment:
I quickly checked the changed files for changes regarding imports: nowhere has been anything referenced to android.support.v4.*

So maybe this can be removed.